tests/performance/test_performance.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+export PYTHONPATH=/home/kbyers/netmiko/tests
+cd /home/vera/netmiko/tests/performance
```

Comment:
Need to get rid of hard-coding to a home directory. We can use ~ for the PYTHONPATH.

I would probably just get rid of the `cd` command and then if either of the test_netmiko.py or gen_graph.py fail add into the failure message that you need to be in {NETMIKO_REPO}/tests/performance directory.